### PR TITLE
Add url encoded N_NODE_MIRROR to the README

### PR DESCRIPTION
**Pull request by BreakBB**
# Pull Request

Adds the information collected in #665 to the README

### CODE REVIEW

File: README.md
Comment (shadowspawn) [inline README.md:214]: The example is not "users in China" so this is not quite what I want. Maybe:

The most common example is from users in China who can define:

**Review by shadowspawn — APPROVED** [review]

File: README.md
Comment (BreakBB) [reply README.md]: That makes sense. I updated the sentence to your suggestion.

**Comment by shadowspawn** [comment]
(Thanks. I expect to merge this into develop next week. Likely to sit around for a while before it lands on master, waiting for some bugs or features to drive a release.)

**Comment by BreakBB** [comment]
Sure thing, nothing urgent from my side. Thanks for keeping me updated! 🙏🏻

**Comment by shadowspawn** [comment]
Released with v7.2.2

**Pull request merged**
